lib/host-sp-messages/src/lib.rs

```diff
@@ -73,15 +91,46 @@ pub enum SpToHost {
     _Unused,
     Ack,
     DecodeFailure(DecodeFailureReason),
-    BootStorageUnit(u8),
+    BootStorageUnit(Bsu),
     Identity {
         model: u8,
         revision: u8,
         serial: [u8; 11], // TODO serial format?
     },
-    Status(u64), // TODO replace u64 with bitflags type?
-    MacAddresses([[u8; 6]; 16]), // TODO send as a data blob? how fixed is 16?
-    RotResponse, // Followed by a binary data blob (the response)
+    MacAddresses {
+        base: [u8; 6],
+        count: u8,
```

Comment:
It seems like it's up in the air whether `count` will be a `u8` or a `u16`, so maybe leave a note here to that effect?

Reply:
Noted in 1da97c8e